proto/cosmos/crisis/v1/tx.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+package cosmos.crisis.v1;
+
+import "cosmos/msg/v1/msg.proto";
+
+// Msg defines the crisis Msg service.
+service Msg {
```

Comment:
How about adding a genesis.proto file to define the initial state? The SDK's DefaultGenesis could be one entry gov == SUPER_ADMIN, and chains could also overwrite in their upgrade handlers.